### PR TITLE
minor doc fixes

### PR DESCRIPTION
**Pull request by aiuto**
- Fixes for links that moved.
- add rule to generate pkg_files docs

### CODE REVIEW

File: pkg/README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 -   [Overview](#overview)
 -   [Roadmap](#roadmap)
 -   [Reference](docs/reference.md)
```
Comment (nacl) [inline pkg/README.md:6]: Should this start with a slash as well?

File: pkg/README.md [hunk 2]
```diff
@@ -21,21 +21,10 @@ and debian package.
...
-    name = "rules_pkg",
-    urls = [
-        "https://mirror.bazel.build/github.com/bazelbuild/rules_pkg/releases/download/0.4.0/rules_pkg-0.4.0.tar.gz",
-        "https://github.com/bazelbuild/rules_pkg/releases/download/0.4.0/rules_pkg-0.4.0.tar.gz",
-    ],
-    sha256 = "038f1caa773a7e35b3663865ffb003169c6a71dc995e39bf4815792f385d837d",
-)
-load("@rules_pkg//:deps.bzl", "rules_pkg_dependencies")
-rules_pkg_dependencies()
-```
```
Comment (nacl) [inline pkg/README.md:36]: It might be worth leaving something here for how to use rules_pkg at HEAD, but it's not essential.

**Review by nacl — APPROVED** [review]
LGTM.  Comments are nitty or non-actionable.

File: pkg/README.md (hunk 1 above)
Comment (aiuto) [reply pkg/README.md:6]: I could, but it would be /pkg/docs/reference.md.  docs is a sibling of this file, examples is a level up

**Pull request merged**